src/plugin/cursor.js

```diff
@@ -200,12 +201,15 @@ export default class CursorPlugin {
                         {
                             display: 'inline',
                             pointerEvents: 'none',
-                            margin: 'auto'
+                            margin: 'auto',
+                            visibility: 'hidden' //initial value will be hidden just for measuring purpose
                         },
                         this.params.customShowTimeStyle
                     )
                 )
             );
+            //initial value to measure display width
+            this.displayTime.innerHTML = '0:00:000';
```

Comment:
instead of hardcoding the time value, use the (user-supplied or default) format time method instead.

Reply:
Oh, I didn't see there is custom format method. Thanks for pointing it, I've updated.